code/modules/antagonists/changeling/powers/mutations.dm

```diff
@@ -292,6 +292,7 @@
 	projectile_type = /obj/projectile/tentacle
 	caliber = CALIBER_TENTACLE
 	icon_state = "tentacle_end"
+	icon = 'icons/obj/weapons/guns/projectiles.dmi'
```

Comment:
have a feeling this one probbbbbbably doesn't actually need an icon, will check later